code/game/objects/items/tools/misc_tools.dm

```diff
@@ -275,6 +275,25 @@
 	desc = "It's an invisible pen marker."
 	pen_colour = "white"
 
+/obj/item/tool/pen/fountain
+	desc = "A luxurious fountain pen, embossed with gold accents. Its intricate mechanics allow the user to switch between various ink colors with a simple twist."
+	name = "fountain pen"
+	icon_state = "fountain_pen"
+	item_state = "fountain_pen"
+	matter = list("metal" = 20, "gold" = 10)
+	var/list/colour_list = list("red", "blue", "green", "yellow", "purple", "pink", "brown", "black", "orange") // Can add more colors as required
+	var/current_colour_index = 1
+
+/obj/item/tool/pen/fountain/attack_self(mob/living/carbon/human/user)
```

Comment:
Overall, i'd prefer this to be a radial over twisting it a gajillion times to get the wanted color

Reply:
I tried to do this yesterday but I figured out that I need to make 9 icons for each color because the "color" variable doesn't seem to work in the radial proc so I can't just use that to make 9 different colored icons out of 1 icon.

I do not have 9 icons to fill the radial at the moment so I won't be able to make this in to a radial in this PR.